addons/website_slides/static/tests/tours/slide_course_publisher_standard.js

```diff
@@ -100,6 +100,48 @@ wTourUtils.registerWebsitePreviewTour('course_publisher_standard', {
 }],
     slidesTourTools.addImageToSection('Introduction', 'Overview', true),
     slidesTourTools.addPdfToSection('Introduction', 'Exercise', true),
+    slidesTourTools.addQuizToSection('Introduction', 'quiz', true),
+[{
+    content: 'eLearning: add a question',
+    trigger: ':iframe input[name="question-name"]',
+    run: "edit Gender",
+}, {
+    content: 'eLearning: enter 1st option',
+    trigger: ':iframe input.o_wslides_js_quiz_answer_value',
+    run: "edit Male",
+}, {
+    content: 'eLearning: enter 2nd option',
+    trigger: ':iframe div.list-group .o_wslides_js_quiz_answer:nth-of-type(2) input',
+    run: "edit Female",
+}, {
+    content: 'eLearning: select a correct option',
+    trigger: ':iframe .o_wslides_js_quiz_icon',
+    run: "click",
+}, {
+    content: 'eLearning: click SAVE & NEW button',
+    trigger: ':iframe .o_wslides_js_quiz_validate_and_add_question',
+    run: "click",
+}, {
+    content: 'eLearning: should be able to add 2nd question',
+    trigger: ':iframe input[name="question-name"]',
+    run: "edit Hobbies",
+}, {
+    content: 'eLearning: enter 1st option',
+    trigger: ':iframe input.o_wslides_js_quiz_answer_value',
+    run: "edit Reading",
+}, {
+    content: 'eLearning: enter 2nd option',
+    trigger: ':iframe div.list-group .o_wslides_js_quiz_answer:nth-of-type(2) input',
+    run: "edit Coding",
+}, {
+    content: 'eLearning: select a correct option',
+    trigger: ':iframe .o_wslides_js_quiz_icon',
+    run: "click",
+}, {
+    content: 'eLearning: save the question',
+    trigger: ':iframe a.o_wslides_js_quiz_validate_question',
+    run: "click",
+}],
```

Comment:
Could we check that we can't enter a third question this time? the opposite of `content: 'eLearning: should be able to add 2nd question',`

Reply:
To check for the presence of an element, we have `isCheck: true`, but I could not come up with a way where we can check for the absence of an element (which in my case was the `+ Add Question` button). :frowning_face: